tests/test_xr_dataset.py

```diff
@@ -0,0 +1,11 @@
+import xarray as xr
+
+
+def test_crs(ds):
+    """TODO: test if 'epsg' or 'proj:espg' in ds.coords or 'crs' in ds.attrs or spatial_ref
+    object in ds.coords/attrs.
+
+    Args:
+        ds (_type_): _description_
+    """
+    raise NotImplementedError
```

Comment:
Well that's just sad. 🥇

Reply:
Fair enough, removed tests directory. because there are no tests yet.

Reply:
Well better to write tests than remove it altogether, but fair enough.

Reply:
Will resolve conversion in order to proceed with merging